ListManipulations/ListIntersection.py

```diff
@@ -0,0 +1,19 @@
+
+list1 = [2,3,4,2,5,6,3,7,7]
+list2 = [1,7,9,0,4,2,3,5]
+newList = []
+
+# Loop through list 1
+# loop through list 2
+# if number in list1 is equal to number in list 2 add it to the new list
+# if number already exists in the new list ignore it
+# Return new list
+
+def list_intersection(lst1, lst2):
+    for i in list1:
+        for s in list2:
```

Comment:
This is very slow considering you will go through list 2 multiple times. There is a way to do this with Sets and with a complexity of O(n). Your current complexity is O(n^2)...I think :). Tip: Use sets.